main/webapp/modules/core/langs/translation-en.json

```diff
@@ -193,7 +193,7 @@
     "core-dialogs/use-this-val": "Use this value",
     "core-dialogs/browse-only-these": "Browse only these values",
     "core-dialogs/browse-this-cluster": "Browse this cluster",
-    "core-dialogs/no-cluster-found": "No clusters were found with the selected method",
+    "core-dialogs/no-cluster-found": "Click Cluster to find clusters on column \"$1\" using the parameters above.",
```

Comment:
I think we need two different messages: one for the initial state of the dialog, where no clusters have been computed yet, and another one for the case where clusters have been computed, but none were found.

Reply:
Yes. Sorry, I checked in a WIP. I have updated it again and it is now complete. Please review. Thanks.